### PR TITLE
Add com.lakoliu.Furtherance

### PR DESCRIPTION
**Pull request by lakoliu**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.lakoliu.Furtherance.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "furtherance",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
```
Comment (hfiguiere) [inline com.lakoliu.Furtherance.json:16]: no. there is a specific dbus name for that, and that what you should use.

File: com.lakoliu.Furtherance.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```
Comment (hfiguiere) [inline com.lakoliu.Furtherance.json:22]: this is not allowed, there is no network during the build. you need to use flatpak-builder-tools to generate a list of crates to download: https://github.com/flatpak/flatpak-builder-tools

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by lakoliu** [comment]
Thank you for your suggestions. I have made the changes as suggested. I believe everything is now in order.

File: com.lakoliu.Furtherance.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                    "type":"script",
+                    "dest-filename":"autogen.sh",
+                    "commands":[
+                        "autoreconf -si"
+                    ]
+                }
+            ]
+        },
+        {
+            "name":"gtk",
```
Comment (hfiguiere) [inline com.lakoliu.Furtherance.json:71]: gtk 4.6.2 is already in the runtime

File: com.lakoliu.Furtherance.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            "sources":[
+                {
+                    "type":"archive",
+                    "url":"https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz",
+                    "sha256":"782d5951fbfd585fc9ec76c09d07e28e6014c72db001fb567fff217fb96e4d8c"
+                }
+            ]
+        },
+        {
+            "name":"libadwaita",
```
Comment (hfiguiere) [inline com.lakoliu.Furtherance.json:87]: libadwaita-1 1.1 is in the runtime

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.lakoliu.Furtherance

**Comment by lakoliu** [comment]
Thank you. I've updated GTk to 4.6.2 and libadwaita to 1.1

File: com.lakoliu.Furtherance.json (hunk 3 above)
Comment (hfiguiere) [reply com.lakoliu.Furtherance.json:71]: what I mean here it to REMOVE IT.

File: com.lakoliu.Furtherance.json (hunk 4 above)
Comment (hfiguiere) [reply com.lakoliu.Furtherance.json:87]: an REMOVE THIS, and the dependencies

**Comment by lakoliu** [comment]
bot, build com.lakoliu.Furtherance

File: com.lakoliu.Furtherance.json (hunk 4 above)
Comment (lakoliu) [reply com.lakoliu.Furtherance.json]: Okay, sorry about that, I've removed both.

**Comment by lakoliu** [comment]
bot, build com.lakoliu.Furtherance

**Pull request closed without merging**